third_party/freedreno/wrap/wrap-util.c

```diff
@@ -106,6 +108,11 @@ void rd_start(const char *name, const char *fmt, ...)
 		 */
 		rd_write_section(RD_GPU_ID, &gpu_id, sizeof(gpu_id));
 	}
+
+	// GOOGLE: Write out the chip id.
+	if(chip_id) {
```

Comment:
should here both be chip_id instead of gpu_id?

Reply:
out of curiosity, why do we add this write_section here? it seems that the latest freedreno change doesn't have this part in this file?